### PR TITLE
Add custom rule for user anderlinil and bucket infn-ai-test

### PR DESCRIPTION
**Pull request by nicolamori**
Allow Lucio to play with a test basket for INFN-AI integration tests.

### CODE REVIEW

File: HERD/http/base/authz/users.rego [hunk 1]
```diff
@@ -30,3 +30,15 @@ allow {
   permissions := data.roles.permissions.user
   permissions[_] == {"action": input.action}
 }
+
+# user anderlinil for bucket infn-ai-test
+allow {
+  grp := input.claims["wlcg.groups"]
+  grp[_] == data.roles.permissions.user_groups[_]
+  username := split(lower(input.claims.preferred_username),"@")[0]
+  username == anderlinil
```
Comment (dciangot) [inline HERD/http/base/authz/users.rego]: need double-quotes

File: HERD/http/base/authz/users.rego [hunk 2]
```diff
@@ -30,3 +30,15 @@ allow {
...
+  input.bucket == infn-ai-test
```
Comment (dciangot) [inline HERD/http/base/authz/users.rego]: need double-quotes

File: HERD/http/base/authz/users.rego (hunk 1 above)
Comment (nicolamori) [reply HERD/http/base/authz/users.rego]: Added in commit 2fff98f.

File: HERD/http/base/authz/users.rego (hunk 2 above)
Comment (nicolamori) [reply HERD/http/base/authz/users.rego]: Added in commit 2fff98f.

**Pull request merged**